extensions-contrib/opentelemetry-emitter/src/main/java/org/apache/druid/emitter/opentelemetry/OpenTelemetryEmitterConfig.java

```diff
@@ -19,9 +19,30 @@
 
 package org.apache.druid.emitter.opentelemetry;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.Collections;
+import java.util.Map;
+
 /**
- * The placeholder for future configurations but there is no configuration yet
+ * The placeholder for configurations
  */
 public class OpenTelemetryEmitterConfig
 {
+  @JsonProperty
+  @NotNull
+  private final Map<String, String> defaultAttributes;
```

Comment:
Should this be called `defaultSpanAttributes` to be clearer?  Also, what do you think about setting these attributes on the `Resource` so you don't have to set it redundantly on every single span?

Reply:
Yeah, good point
I didn't know about `Resource`, let me play with that

Reply:
Actually, we may be able to avoid any code change and use environment variables to set this.  Are we using [autoconfigure](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md) in the druid extension?  If so you could just set the environment variable in the helm deployment.

```
OTEL_RESOURCE_ATTRIBUTES=exporter.trace.kafka=exclude
```

Reply:
Yeah, we're using autoconfigure in our extension
Let's do that